gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/model/GbUser.java

```diff
@@ -17,43 +17,46 @@
 
 import java.io.Serializable;
 
+import lombok.experimental.Accessors;
```

Comment:
This name of this package makes me worry...

Reply:
I understand, but if lombok remove it, we can implement this another way. Being able to chain ops is a nice feature.